src/java.base/share/classes/java/lang/foreign/FunctionDescriptor.java

```diff
@@ -25,13 +25,13 @@
 
 package java.lang.foreign;
 
+import jdk.internal.foreign.FunctionDescriptorImpl;
+
 import java.lang.invoke.MethodHandle;
 import java.lang.invoke.MethodType;
+import java.util.List;
 import java.util.Objects;
 import java.util.Optional;
-import java.util.List;
-
-import jdk.internal.foreign.FunctionDescriptorImpl;
```

Comment:
Just curious where this style/convention is coming from, I've always put internal packages after the import of standard APIs.

Reply:
Both alternatives were present and I picked the most prevailing one in the package. Also, this is what you get when you auto format in IntelliJ. But if there is a preference for the other way, we could switch. No problem.

Reply:
I don't have any strong  opinion on the topic, I was mostly just curious when I saw "conformant" in the title.